frontend/app/src/store/main.ts

```diff
@@ -777,7 +777,7 @@ export class MainStore {
     // if we don't pass the params, we should use previous params for invalidate query
     const query2 = this.appendQueryParams(
       'gobounties/all',
-      queryLimit,

```

Comment:
a simpler change would to be to just increase this queryLimit

Reply:
yeah i thought of doing that. but I assumed there was a reason that it was made 100 initially. So i went with the route of if a query limit is passed that should be used and if not it default to the 100 @kevkevinpal

Reply:
@kevkevinpal  still waiting for a review